e2e/specs/st_dataframe_sizes.spec.js

```diff
@@ -15,38 +15,45 @@
  * limitations under the License.
  */
 
-describe("Dataframes", () => {
-  const DF_SELECTOR = ".stDataFrame";
-  const TABLE_SELECTOR = ".stTable > table";
-
+describe("Dataframes and Tables snapshots", () => {
   before(() => {
+    // Increasing timeout since we're waiting for
+    // dataframes and tables to be rendered.
+    Cypress.config("defaultCommandTimeout", 30000);
```

Comment:
My first thought was do we really need 30 seconds for 17 tables with 2 rows and 4-20 columns but then realized Arrow should be addressing the performance issue (?). Perhaps this is more for the arrow implementation but we many need to define what our performance expectations are when defining the timeouts?

Reply:
> but we may need to define what our performance expectations are when defining the timeouts?

Agree